Formula/n/netpbm.rb

```diff
@@ -35,6 +36,10 @@ class Netpbm < Formula
 
   conflicts_with "jbigkit", because: "both install `pbm.5` and `pgm.5` files"
 
+  resource "html" do
+    url "https://svn.code.sf.net/p/netpbm/code/userguide", revision: "5023"
+  end
```

Comment:
How do we know which `revision` to use?

Reply:
As the source USERDOC says:
```
This manual is always up to date.  It is not maintained on a release
schedule like the source code is, but rather updated continuously.
The user manual describes past Netpbm function as well as the present,
so you can use the current manual with old Netpbm code.
```
I'm still trying to figure out a livecheck for the `userguide` repo, as there's no obvious equivalent to the `Release` strings in the source repo.

Reply:
OK, I've figured out the proper livecheck, but the output highlights a potential issue:
```
% brew livecheck -r netpbm
netpbm: 11.02.12 ==> 11.02.12
  html: 11.02.12 ==> 5015
```
Should I manually declare `version "5015"` in the resource block?

Reply:
OK, added `version "5015"` and explanatory comments.